src/Cosmonaut/Cosmonaut.csproj

```diff
@@ -29,6 +29,11 @@
   <ItemGroup>
     <PackageReference Include="Humanizer.Core.uk" Version="2.5.16" />
     <PackageReference Include="Microsoft.Azure.DocumentDB.Core" Version="2.2.1" />
+    <PackageReference Include="QueryInterceptor.Core" Version="1.0.7" />
```

Comment:
Do we need to pull in QueryInterceptor.Core for this? Had a look at the library and it doesn't look that there is a lot of code in there to justify the dependency. I'd rather have the code pulled into or just keep what I really need from it.

Reply:
Resolved